brain_games/games/brain_progression.py

```diff
@@ -0,0 +1,36 @@
+from brain_games.engine import play
+from random import randint
+
+
+def gen_progression():
+    num_2 = randint(1, 20)
+    num_1 = randint(1, 100)
+    count = 1
+    ls = []
```

Comment:
именуй более понятно
```suggestion
    progression = []
```